notes.md

```diff
@@ -192,7 +192,7 @@ For separate tracks, Matroska needs to define exactly which track does what. `Tr
 
 The 3D support is still in infancy and may evolve to support more features.
 
-The StereoMode used to be part of Matroska v2 but it didn't meet the requirement for multiple tracks. There was also a bug in libmatroska prior to 0.9.0 that would save/read it as 0x53B9 instead of 0x53B8. `Matroska Readers` may support these legacy files by checking Matroska v2 or 0x53B9. The [older values](http://www.matroska.org/node/1/revisions/74/view#StereoMode) were 0: mono, 1: right eye, 2: left eye, 3: both eyes.
+The StereoMode used to be part of Matroska v2 but it didn't meet the requirement for multiple tracks. There was also a bug in libmatroska prior to 0.9.0 that would save/read it as 0x53B9 instead of 0x53B8. `Matroska Readers` may support these legacy files by checking Matroska v2 or 0x53B9. The older values were 0: mono, 1: right eye, 2: left eye, 3: both eyes.
```

Comment:
I was worried about this removal but it seems [even in 2010](https://web.archive.org/web/20120326131802/http://www.matroska.org/node/1/revisions/74/view) these values were the ones in use.

Reply:
I had checked the link before removing it, and that node didn't exist anymore in Drupal. Hence the link was already dead, and I couldn't find an active page anywhere else in Drupal describing older values.